tests/phpunit.xml.dist

```diff
@@ -68,5 +68,9 @@
     </testsuites>
     <php>
         <env name="MONGODB_SERVER" value="mongodb://mongodb:27017"/>
+        <env name="SYMFONY_DEPRECATIONS_HELPER" value="max[direct]=1"/> <!-- Remove it when QueryAnalyzer is removed -->
```

Comment:
If we must ignore a specific deprecation message, maybe we could use the [baseline](https://symfony.com/doc/current/components/phpunit_bridge.html#baseline-deprecations) for this purpose.

Reply:
I didn't know about that feature, thanks! I've tried to remove the deprecation in https://github.com/doctrine-extensions/DoctrineExtensions/pull/2397